### PR TITLE
Added Lastday

### PR DESCRIPTION
**Pull request by madisonlong1**
## Overview
Updated back end so lastday exists and works for testing, we used the tests below to confirm. As no front end changes have occurred yet, we have no screenshots an no validation, this is coming soon 

UPDATE: 
Now contains front end changes as well, see deployment below: 
https://happycows-madisonlong1.dokku-03.cs.ucsb.edu/

image

## Tests

- [x] Backend Unit tests (`mvn test`) pass
- [x] Backend Test coverage (`mvn test jacoco:report`) 100%
- [x] Backend Mutation tests (`mvn test pitest:mutationCoverage`) 100% 
- [x] Frontend Unit tests (`npm test`) pass
- [x] Frontend Test coverage (`npm run coverage`) 100%
- [x] Frontend Mutation tests (`npx stryker run`) 100% 
- [x] Frontend Linting (`npx eslint --fix src`) 

## Linked Issues

Closes #0

### CODE REVIEW

**Comment by KarimBenyassine1** [comment]
Last day must be added in the form in frontend, and adjusted on Create and Update pages.

File: frontend/src/main/components/Nav/AppNavbar.js
Comment (pconrad) [inline frontend/src/main/components/Nav/AppNavbar.js]: is this comment needed?

File: frontend/src/main/components/Nav/AppNavbar.js
Comment (pconrad) [inline frontend/src/main/components/Nav/AppNavbar.js:15]: It might be helpful to revert this change and the other changes to frontend code so that you don't have to wait for frontend CI/CD jobs to run on this PR; it will speed things up a lot!

**Comment by pconrad** [comment]
> Last day must be added in the form in frontend, and adjusted on Create and Update pages.

True, but that can be done in a separate issue. It's helpful to get this into the code base first and then tackle the frontend separately, as long as merging it doesn't break the app (which should be tested on a dokku qa deployment before we merge.

@madisonlong1 can you deploy this to a qa branch and include a link to that in the description?

File: frontend/src/main/components/Nav/AppNavbar.js
Comment (madisonlong1) [reply frontend/src/main/components/Nav/AppNavbar.js]: No, it was a test at the time but now it serves no purpose, I will remove it!

**Review by pconrad — DISMISSED** [review]
Please see: https://ucsb-cs156.github.io/topics/pull_requests/package_lock_json.html

File: frontend/src/main/components/Commons/CommonsForm.js
Comment (pconrad) [inline frontend/src/main/components/Commons/CommonsForm.js:315]: You may want to consider using the `<Row><Col>` construct that you find earlier in the file to get the Start Date and End Date side by side instead of on top of one another.  That would make the form look a bit more compact.

Comment (madisonlong1) [reply frontend/src/main/components/Commons/CommonsForm.js:315]: Good Idea! the change has been implemented

**Comment by madisonlong1** [comment]
> Please see: https://ucsb-cs156.github.io/topics/pull_requests/package_lock_json.html

The requested change was pushed

**Review by livjono — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM!

**Pull request merged**
